### PR TITLE
LSR-1050

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

File: android/library/build.gradle [hunk 1]
```diff
@@ -22,7 +22,10 @@ android {
 			includeAndroidResources = true
 		}
 	}
-
+	compileOptions {
+		sourceCompatibility JavaVersion.VERSION_1_8
+		targetCompatibility JavaVersion.VERSION_1_8
+	}
```
Comment (victorg1991) [inline android/library/build.gradle]: Is this safe? no bugs involved? :P

File: android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java [hunk 2]
```diff
@@ -153,33 +155,48 @@ public static void relogin(LoginListener loginListener) {
...
-						User user = new User(jsonObject);
-						currentUser = user;
-						loginListener.onLoginSuccess(user);
-					} else {
-						SessionContext.logout();
-						loginListener.onLoginFailure(new AccessControlException("User invalid"));
-					}
-				} catch (Exception e) {
-					loginListener.onLoginFailure(e);
+		Executor.execute(() -> {
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java]: Where is my boilerplate? :P

File: android/library/build.gradle (hunk 1 above)
Comment (victorg1991) [inline android/library/build.gradle]: Is this safe? :O

**Comment by victorg1991** [comment]
LGTM 👍 We need to make a release with this, right? Maybe is better to merge this with master?

File: android/library/build.gradle (hunk 1 above)
Comment (nhpatt) [reply android/library/build.gradle]: OFC! :D

**Comment by nhpatt** [comment]
No, until we know for sure that is causing problems with our client in LESA

**Pull request closed without merging**